mobile/test/java/org/chromium/net/CronetHttp3Test.java

```diff
@@ -51,6 +53,10 @@ public class CronetHttp3Test {
   private CronvoyLogger logger;
   // The engine for this test.
   private CronvoyUrlRequestContext cronvoyEngine;
+  // The engine builder for this test.
```

Comment:
This looks unused given that we are creating a local instance in L95.

Reply:
fixed